convert2rhel/systeminfo.py

```diff
@@ -549,5 +500,26 @@ def is_systemd_managed_service_running(service):
     return running
 
 
+def generate_rpm_va(log_filename=PRE_RPM_VA_LOG_FILENAME):
+    """RPM is able to detect if any file installed as part of a package has been changed in any way after the
+    package installation.
+
+    Here we are getting a list of changed package files of all the installed packages. Such a list is useful for
+    debug and support purposes. It's being saved to the default log folder as log_filename."""
+    if tool_opts.no_rpm_va:
+        logger.info("Skipping the execution of 'rpm -Va'.")
+        return
+
+    logger.info(
+        "Running the 'rpm -Va' command which can take several"
+        " minutes. It can be disabled by using the"
+        " --no-rpm-va option."
+    )
+    rpm_va, _ = utils.run_subprocess(["rpm", "-Va"], print_output=False)
+    output_file = os.path.join(LOG_DIR, log_filename)
+    utils.store_content_to_file(output_file, rpm_va)
+    logger.info("The 'rpm -Va' output has been stored in the %s file." % output_file)
+
```

Comment:
To minimize the changes to the systeminfo that doesn't seem related to the code, why don't we keep this inside the class and only call system_info.generate_rpm_va() inside the action classs? That would minimize all of those logger changes

Reply:
To expand a bit on this, I think this is a good change, moving the logger instance to be outside the class is definitely a way to go. But it also feels unrelated to the scope of the ticket. We could open a new PR that patches this as well.

Reply:
Okay. I wanted to take the `generate_rpm_va` outside the class. But you are right, this is unrelated to the ticket. I'll open a new PR with the changes.